frontend/js/src/tags/TagsComponent.tsx

```diff
@@ -311,15 +311,11 @@ export default function AddTagSelect(props: {
             flexWrap: "nowrap",
             overflowX: "auto",
             paddingRight: "3.5em",
+            scrollbarWidth: "none",
```

Comment:
I think the addition of `scrollbarWidth: "thin"` is all we would need to resolve the issue for Firefox (reverting the webkit-specific styles)